README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
+- Should be stable: no random failure
+- Require a setup with at least 2 nodes
+- Independent from the backends and the network:
+  - backends and network must be started separately
+  - backends and network can be located locally / cloud, started through docker-compose / kubernetes
+  - tests can be run multiple times without restarting backends / network
+- Easy to configure through settings files
+- Each test must not finished before all the created tuples have been executed
```

Comment:
Just an English type: finished => finish

Or, to go further, what do you think of the following?

```suggestion
- Each test must not finished before all the created tuples have been executed
+ Should not complete before all the tuples it created have been executed
```

Reply:
👍